CauldronMods/Controller/Heroes/LadyOfTheWood/Cards/FallCardController.cs

```diff
@@ -14,31 +14,27 @@ public override void AddTriggers()
 		{
 			//Whenever LadyOfTheWood deals lightning damage to a target, reduce damage dealt by that target by 1 until the start of your next turn.
 			Func<DealDamageAction,bool> criteria = (DealDamageAction dd) => dd.DamageSource != null && dd.DamageSource.IsSameCard(base.CharacterCard) && dd.DamageType == DamageType.Lightning;
```

Comment:
The damage-type criteria should not be in this location, because the AddStatusEffectToDamage triggertype is really early in the trigger resolution order, and this will likely cause misapplications or missed applications if there's a damage-type changing effect. If you want to see for yourself, you could try tests with Megalopolis's Cramped Quarters Combat and/or Visionary's Twist the Ether